OracleWebCenterSites/dockerfiles/12.2.1.4/sites-container-scripts/startSitesServer.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Description: This script is used to start a Sites Managed Server.
+#
+export DOMAIN_NAME=$DOMAIN_NAME
+export DOMAIN_ROOT="/u01/oracle/user_projects/domains"
+export DOMAIN_HOME="${DOMAIN_ROOT}/${DOMAIN_NAME}"
+export SITES_CONTAINER_SCRIPTS=/u01/oracle/sites-container-scripts
+
+. $DOMAIN_HOME/servers/${SITES_SERVER_NAME}/logs/param.properties
+WCSITES_OLD_MANAGED_HOSTNAME=$WCSITES_MANAGED_HOSTNAME
+
+#=================================================================
+function _int() {
+   echo "INFO: Stopping container."
+   echo "INFO: SIGINT received, shutting down Managed Server!"
+   $DOMAIN_HOME/bin/stopManagedWebLogic.sh $SITES_SERVER_NAME "t3://"$WCSITES_ADMIN_HOSTNAME:$WCSITES_ADMIN_PORT
+   exit;
+}
+#=================================================================
+function _term() {
+   echo "INFO: Stopping container."
+   echo "INFO: SIGTERM received, shutting down Managed Server!"
+   $DOMAIN_HOME/bin/stopManagedWebLogic.sh $SITES_SERVER_NAME "t3://"$WCSITES_ADMIN_HOSTNAME:$WCSITES_ADMIN_PORT
+   exit;
+}
+#=================================================================
+function _kill() {
+   echo "INFO: SIGKILL received, shutting down Managed Server!"
+   $DOMAIN_HOME/bin/stopManagedWebLogic.sh $SITES_SERVER_NAME "t3://"$WCSITES_ADMIN_HOSTNAME:$WCSITES_ADMIN_PORT
+   exit;
+}
+
+#=================================================================
+#== MAIN Starts here...
+#=================================================================
+trap _int SIGINT
+trap _term SIGTERM
+trap _kill SIGKILL
+
+WCSITES_MANAGED_HOSTNAME=`hostname -i`
+
+#replaceWith=$WCSITES_MANAGED_HOSTNAME
```

Comment:
Don't commit commented out code. Just remove it. It'll be retained in git history if you need to go back and see what it was.